index.html

```diff
@@ -57,5 +59,204 @@ <h1>
         </ul>
       </main>
     </div>
+
+    <!-- Toolbar-Headline Section Ends -->
+
+    <!-- Work Section Starts -->
+
+    <!-- Project 1 -->
+
+    <div class="work-section">
+      <div class="snapshot">
+        <img src="./images/snapshot-1.svg" alt="" />
+      </div>
+      <div>
+        <p class="snapshot-title">Tonic</p>
+        <div class="client-info">
+          <p class="client">CANOPY</p>
+          <div class="circle"></div>
+          <p>Back End Dev</p>
+          <div class="circle"></div>
+          <p>2015</p>
+        </div>
+        <p class="project-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="used-language">
+          <div class="language">html</div>
+          <div class="language">css</div>
+          <div class="language">javascript</div>
+        </div>
```

Comment:
- I believe it will be better for our `class="used-language"` to be in a `<ul>` tag with each language in `<li>` tags. It's important for us to reduce the use of `<divs>` in our code base. 
Can we fix this?? 🤗 

⚠️ Kindly replicate this changes in other project cards as well.

Reply:
Thanks mate! Done!